nasa-gcn/gcn.nasa.gov
# Circulars: Open Automatically Formatted Hyperlinks in New Tab

# Description
This makes any plaintext hyperlinks open in a new tab rather than opening in the same tab.

However, this currently does not do the same for tns, doi, arxiv, or other circular links formatted by remark-rehype-astro. It might make sense to make this same change for tns, doi, and arxiv links but retain the current behavior for automatic circular links since that is intrasite navigation.

# Related Issue(s)
Resolves #2389  

# Testing
1. Navigate to any Circular that contains a URL
2. Click on the URL. This should now open it in a new tab rather than the same tab
